### PR TITLE
[Blueprint] Building an Egress Gateway using Netmaker

### PR DESCRIPTION
**Pull request by v-ctiutiu**
## Overview

- Introducing Netmaker for creating and managing private networks.
- Exploring the Netmaker server UI and capabilities (most important ones for the current guide).
- Floating IP configuration for the Egress GW.
- Steps for using Netmaker as an Egress GW for DOKS clusters.

### CODE REVIEW

**Comment by vladciobancai** [comment]
The following section [Configuring Netmaker Server ](https://github.com/digitalocean/container-blueprints/tree/v-ctiutiu/issue21/netmaker-egress-gateway#configuring-netmaker-server) contains the same steps like the following ones:
[Exploring the Networks Section](https://github.com/digitalocean/container-blueprints/tree/v-ctiutiu/issue21/netmaker-egress-gateway#exploring-the-networks-section), [Exploring the Access Keys Section](https://github.com/digitalocean/container-blueprints/tree/v-ctiutiu/issue21/netmaker-egress-gateway#exploring-the-access-keys-section), [Exploring the Nodes Section](https://github.com/digitalocean/container-blueprints/tree/v-ctiutiu/issue21/netmaker-egress-gateway#exploring-the-nodes-section)  they should be merged into one chapter

File: netmaker-egress-gateway/README.md [hunk 1]
```diff
@@ -0,0 +1,820 @@
...
+
+Looking at the above, you can also notice that the main Netmaker server is part of your private network, and that it's acting as an Egress Gateway (denoted by the green pass mark in the egress column).
+
+**Hint:**
+
+It may happen for some  `Netclients` to not succeed in joining your private network. What you can do in this case is to hit the `Sync` button on the top right corner:
+
+![Healthy](assets/images/doks_network_nodes_sync.png)
+
+If the above still doesn't solve this issue, you can ultimately kill the respective pods. The DaemonSet will spawn fresh ones, and the join process should start again (nodes communicate with master node, meaning the Netmaker server, via GRPC calls).
```
[the remark this answers is not in the document]
Comment (v-ctiutiu) [reply netmaker-egress-gateway/README.md]: Should be fixed now.

**Comment by v-ctiutiu** [comment]
> The following section [Configuring Netmaker Server ](https://github.com/digitalocean/container-blueprints/tree/v-ctiutiu/issue21/netmaker-egress-gateway#configuring-netmaker-server) contains the same steps like the following ones: [Exploring the Networks Section](https://github.com/digitalocean/container-blueprints/tree/v-ctiutiu/issue21/netmaker-egress-gateway#exploring-the-networks-section), [Exploring the Access Keys Section](https://github.com/digitalocean/container-blueprints/tree/v-ctiutiu/issue21/netmaker-egress-gateway#exploring-the-access-keys-section), [Exploring the Nodes Section](https://github.com/digitalocean/container-blueprints/tree/v-ctiutiu/issue21/netmaker-egress-gateway#exploring-the-nodes-section) they should be merged into one chapter

Should be fixed now.

**Comment by vladciobancai** [comment]
looks good to me 👍

**Comment by v-ctiutiu** [comment]
Leaving Netmaker aside for now.

**Pull request closed without merging**